tools/firmware-utils/src/jcgimage.c

```diff
@@ -387,8 +400,8 @@ main(int argc, char **argv)
 		sizeo = sizeof(*jh) + sizeu;
 	}
 
-	if (sizeo > MAXSIZE) {
-		errx(1,"payload too large: %zd > %zd\n", sizeo, MAXSIZE);
+	if (sizeo > maxsize) {
+		errx(1,"payload too large: %zd > %zd\n", sizeo, maxsize);
```

Comment:
ERROR: space required after that ',' (ctx:VxV)
    #88: FILE: tools/firmware-utils/src/jcgimage.c:404:
    +               errx(1,"payload too large: %zd > %zd\n", sizeo, maxsize);
      
Otherwise I think this looks good.